projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -36,54 +44,86 @@ export function clickOnRequestQuoteInCartAndExpectQuotePage(): void {
  * Clicks on 'Request Quote' on the cart page.
  */
 export function clickOnRequestQuoteInCart(): void {
+  log(
```

Comment:
Could you, please, remove `clickOnRequestQuoteInCart()`. 
This `clickOnRequestQuoteInCartAndExpectQuotePage()` does the same and verifies whether  the quote page is displayed.